css/index.css

```diff
@@ -1,3 +1,356 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+
+html {
+    font-size: 62.5%;
+}
+
+.container{
+    width: 100%;
+	max-width: 880px;
+	margin: 0px auto;
+	/* padding: 25px; */
+}
+
+p {
+    font-size: 1.6rem;
+}
+
+nav {
+	width: 100%;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	margin: 60px 0px 60px 0px;
+	
+}
+
+.nav {
+    width: 73.8%;
+    display: flex;
+    justify-content: space-between;
+    /* border: solid black 1px; */
+}
+
+a {
+	text-decoration: none;
+    color: grey;
+    font-size: 1.6rem;
+	transition-duration: 0.4s;
+}
+
+a:hover {
+	color: black;
+
+}
+
+header {
+	width: 100%;
+	display: flex;
+	justify-content: space-between;
+	/* align-items: center; */
+    margin: 0px 0px 50px 0px;
+    padding: 0;
+}
+
+h1 {
+    font-size: 7.5rem;
+	/* display: flex; */
+}
+
+h2 {
+    font-size: 1.8rem;
+}
+
+.innov {
+	display: flex;
+	flex-direction: column;
+	/* border: solid black 1px; */
+    align-items: center;
+	margin: 10px 0px 0px 50px;
```

Comment:
this margin is making the .innov be 50 px off-center in tablet and mobile-view. How can we fix this?